tests/app/webauthn/test_rest.py

```diff
@@ -0,0 +1,173 @@
+import uuid
+from unittest.mock import ANY
+
+from tests.app.db import create_user, create_webauthn_credential
+
+
+def test_get_webauthn_credentials_returns_all_credentials_for_user(admin_request, notify_db_session):
+    me = create_user(email='a')
+    other = create_user(email='b')
+    first = create_webauthn_credential(me, '1')
+    create_webauthn_credential(me, '2')
+    create_webauthn_credential(other, '3')
+
+    resp = admin_request.get(
+        'webauthn.get_webauthn_credentials',
+        user_id=me.id,
+    )
+
+    creds = sorted(resp['data'], key=lambda x: x['name'])
+    assert len(creds) == 2
+
+    assert creds[0] == {
```

Comment:
not sure - we cant return whole model since the serialize function is how that gets into it. do you mind digging into this?

Reply:
@CrystalPea these tests weren't being run because there was an error importing this test file. If you add an `__init__.py` file in this same directory it fixes the set-up error and this test then fails.

Reply:
ahh, sweet, thanks for solving this @klssmith 💖